### PR TITLE
Implement Profile Photo page component

### PR DESCRIPTION
**Pull request by michaeldotdev**
### What this PR does (required):
Component for User to update and delete a profile photo.

### Screenshots / Videos (required):
[image](https://i.imgur.com/iS4YxVv.png)

### Any information needed to test this feature (required):
Login with a test account, and the card should display on the Dashboard page. 

### Any issues with the current functionality (optional):
N/A

### CODE REVIEW

**Comment by ezeilo-su** [comment]
There is no description in this PR. Please provide more information about this PR.

**Comment by sundayezeilo** [comment]
Please provide more information. Add screenshot(s).

File: client/src/pages/Dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -29,6 +32,8 @@ export default function Dashboard(): JSX.Element {
 
   return (
     <Grid container component="main" className={`${classes.root} ${classes.dashboard}`}>
+      {/* Temp Render */}
```
Comment (rajivtitus) [inline client/src/pages/Dashboard/Dashboard.tsx:35]: 👍🏽

**Comment by rajivtitus** [comment]
> Please provide more information. Add screenshot(s).

Hey Sunday, there is a picture in the PR template under 'Screenshots' :)

File: client/src/components/ProfileSettings/ProfilePhoto/ProfilePhoto.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { Avatar, Box, Button, Paper } from '@material-ui/core';
+import DeleteForeverIcon from '@material-ui/icons/DeleteForever';
+import Typography from '@material-ui/core/Typography';
+import useStyles from './useStyles';
+
+export default function ProfilePhoto(): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    //TODO - Create reuseable Paper&Box Component
```
Comment (rajivtitus) [inline client/src/components/ProfileSettings/ProfilePhoto/ProfilePhoto.tsx:9]: 👍🏽

File: client/src/components/ProfileSettings/ProfilePhoto/ProfilePhoto.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import { Avatar, Box, Button, Paper } from '@material-ui/core';
+import DeleteForeverIcon from '@material-ui/icons/DeleteForever';
+import Typography from '@material-ui/core/Typography';
```
Comment (rajivtitus) [inline client/src/components/ProfileSettings/ProfilePhoto/ProfilePhoto.tsx]: You can remove this import here and add it as part of the import on Line 1 :)

**Review by rajivtitus — APPROVED** [review]
UI looks great! Good use of MUI and nice to see comments about TODO items as well :)

Note: In the future, you can just drag/drop images onto the PR templates in Github.

File: client/src/components/ProfileSettings/ProfilePhoto/ProfilePhoto.tsx (hunk 3 above)
Comment (michaeldotdev) [reply client/src/components/ProfileSettings/ProfilePhoto/ProfilePhoto.tsx]: Sounds good! Will update and merge

**Pull request merged**